_data/simple-icons.json

```
@@ -610,6 +610,11 @@
             "hex": "A81D33",
             "source": "https://www.debian.org/logos"
         },
+        {
+            "title": "Deepin",
```

Fully agree @davidklebanoff, if this is changed this PR is good to go @JoaoPedroMoraes :wink: 

```suggestion
            "title": "deepin",
```